bentoml/deployment/sagemaker/__init__.py

```diff
@@ -291,7 +291,7 @@ def apply(self, deployment_pb, yatai_service, prev_deployment=None):
             sagemaker_client = boto3.client('sagemaker', sagemaker_config.region)
 
             with TempDirectory() as temp_dir:
-                sagemaker_project_dir = os.path.jon(
+                sagemaker_project_dir = os.path.join(
```

Comment:
Was going to open an issue on this, great to see this fixed :+1:

Reply:
Yeah. My b on that one